content/applications/services/helpdesk/advanced/close_tickets.rst

```diff
@@ -15,23 +15,22 @@ As work on a ticket progresses, it is moved along to the next stage in the pipel
 is solved, the ticket is moved to a *folded* stage. This marks the ticket as *closed*.
 
 To fold a stage, navigate to the :menuselection:`Helpdesk` dashboard and click on a team to open the
-pipeline. Hover over a stage's heading, and then click the gear icon that appears in the top-right
-corner of that stage's kanban column.
+pipeline. Hover over a stage's heading, and then click the :guilabel:`⚙️ (gear)` icon that appears
+in the top-right corner of that stage's Kanban column.
 
 .. image:: close_tickets/closing-edit-stage-gear.png
    :align: center
    :alt: View of stage on Helpdesk pipeline with emphasis on gear icon and edit stage option.
 
 .. warning::
-   Clicking the gear icon also displays the option to :guilabel:`Fold` the stage. This setting folds
-   the stage *temporarily* to simplify the kanban view. This does *not* close the tickets in this
-   stage. It also does not permanently fold the stage. If a stage needs to be folded so the tickets
-   can be marked as closed, continue following the steps below.
+   Clicking the :guilabel:`⚙️ (gear)` icon also displays the option to :guilabel:`Fold` the stage.
+   This setting folds the stage *temporarily* to simplify the Kanban view. This does **not** close
+   the tickets in this stage. It also does **not** permanently fold the stage. If a stage needs to
+   be folded so the tickets can be marked as closed, continue following the steps below.
```

Comment:
Just so this is not confusing, I would put this warning after the next part. So complete the steps to make it a folded stage, then have the warning about the temporary folding.